### PR TITLE
Eliminate the fixed model from lxd integration tests

### PR DESCRIPTION
**Pull request by gtrkiller**
Having the model "testing" as the fixed model name causes problems with the integration tests when lxd is the provider. Since for this tests it is not useful to have a fixed model name, I'd attempt to remove it for lxd in order for the tests to pass.

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: .github/workflows/integration_test_run.yaml [hunk 1]
```diff
@@ -252,6 +253,31 @@ jobs:
...
+
+          series=""
+          if [ ! -z ${{ matrix.series }} ]; then
+            series="--series ${{ matrix.series }}"
+          fi
+          module=""
+          if [ ! -z ${{ matrix.modules }} ]; then
+            module="-k ${{ matrix.modules }}"
+          fi
+          tox -e integration -- --keep-models $series $module $args ${{ inputs.extra-arguments }}
```
Comment (gregory-schiano) [inline .github/workflows/integration_test_run.yaml:278]: Isn't there a way to do a bit of factoring between this step and the previous one as 90% of the run script is similar ?
Like having a models_arg output for both steps
And a new step that does all the args/module/series and executes tox using the output of the previous step

Comment (gtrkiller) [reply .github/workflows/integration_test_run.yaml:278]: Not that I know of, but I'll take a look and ask around

Comment (arturo-seijas) [reply .github/workflows/integration_test_run.yaml:278]: I can do that as part of #145 if that works for you. I'll have to resolve the conflict that will arise as part of this anyway

**Review by arturo-seijas — DISMISSED** [review]
Nice!

**Review by weiiwang01 — DISMISSED** [review]

**Review by arturo-seijas — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Pull request merged**